src/04_printing.py

```diff
@@ -18,4 +18,6 @@
  
 print ("x is {}, y is {}, z is {}".format(x, yRound, z)) 
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+# Finally, print the same thing using an f-string
+
+print (f"x is {x}, y is {yRound}, z is {z}") 
```

Comment:
There was typo in this module. The last printing exercise should be like the following:

```
# Using %-formatting to print the same thing

print('x is %d, y is %.2f, z is %s' %(x, round(y, 2), z))
```